app/views/users/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Users page here</h1>
\ No newline at end of file
+<div class="container">
+  <h1 class="mt-5">Users list page</h1>
+
+  <div class="row">
+    <% @users.each do |u| %>
+      <div class="col-md-12 mb-4">
+        <div class="row">
+          <div class="col-md-3">
+            <div class="photo">
+              <% if u.photo.present? %>
+                <img src="<%= u.photo %>" alt="<%= u.name %>" class="img-fluid">
+              <% else %>

```

Comment:
- [ ] Kindly ensure you follow the given wireframe for the user index page so as to meet the requirements

Your design
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/cc6cf95a-d38c-450c-87f5-2e078717cad2)

Given wireframe
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/e91da433-2e9a-4d52-a361-2328fe4ff402)

Reply:
- [ ] Also consider implementing this page so that clicking on a post takes the user to an individual post post page as shown in the wireframe below

![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/68345c53-72be-4e1d-8141-9593285999b5)

Reply:
- [ ] As requested here kindly fix your design so as to follow the wireframe design

Your design
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/8df2f0b3-d6ed-4eec-ade8-66aa0e335701)

Given wireframe
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/832cd1f7-7a5b-4409-bc52-6e05262157b8)

### Also note that clicking on a post should take the user to that post index page as shown below
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/81932478/3fb73046-4f85-4567-adda-f52f4e895f4f)